blocks/footer/footer.css

```diff
@@ -9,23 +9,56 @@ footer .block {
   padding-right: var(--spacing-s);
 }
 
+footer .footer h3 {
+  font-size: var(--body-font-size-xs);
+  font-weight: var(--font-weight-bold);
+  margin: 0 0 10px;
+}
+
 footer .footer p {
   margin: 0;
 }
 
-.columns > div {
-  display: flex;
-  flex-direction: column;
+footer .columns p {
+  max-width: 240px;
+  color: rgb(0 0 0 / 55%);
```

Comment:
should we standardize on using hex or rgb? could also be expressed as `0000008C`

Reply:
IMO its easier to read in rgb when there is an alpha channel. But generally I have no strong opinion on it